keps/sig-node/20190226-pod-overhead.md

```diff
@@ -113,9 +135,9 @@ pod overhead is accountable. If the pod specifies a resource limit, and all cont
 pod specify a limit, then the sum of those limits becomes a pod-level limit, enforced through the
 pod cgroup.
 
-Users are not expected to manually set the pod resources; if a runtimeClass is being utilized,
-the manual value will be discarded. See RuntimeController for the proposal for setting these
-resources.
+Users are not expected to manually set `Overhead`; any prior values will be discarded during admission. If runtimeClass
```

Comment:
@derekwaynecarr had good feedback on initial PR re: using a suberesource, similar to how binding is handled, in order to make sure the end user wouldn't be setting this. Curious about your thoughts, @tallclair